src/utils/api.js

```diff
@@ -0,0 +1,68 @@
+const API_BASE_URL =
+  process.env.NODE_ENV === "production"
+    ? "https://api.articlelist.ignorelist.com"
+    : "http://localhost:3001";
+
+// Fetch articles by a user
+export const getArticlesByUser = async (token) => {
+  try {
+    const response = await fetch(`${API_BASE_URL}/articles`, {
+      method: "GET",
+      headers: { Authorization: `Bearer ${token}` },
+    });
+    if (!response.ok) {
+      throw new Error("Network response was not ok");
+    }
+    const articles = await response.json();
+    return Array.isArray(articles) ? articles : [];
+  } catch (error) {
+    console.error("Error fetching articles:", error);
+    return []; // Return an empty array in case of error
+  }
+};
+
+// Create a new article
+export const createArticle = async (articleData, token) => {
+  try {
+    const response = await fetch(`${API_BASE_URL}/articles`, {
+      method: "POST",
+      headers: {
+        Authorization: `Bearer ${token}`,
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(articleData),
+    });
+    if (!response.ok) {
+      throw new Error("Network response was not ok");
+    }
+    return await response.json();
+  } catch (error) {
```

Comment:
It’s better not to use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request and this end is in `index.js`  after all blocks `then`. And in real projects blocks `catch` usually open error popups and so on.